helper-npm-install-peer-deps

```diff
@@ -34,7 +34,7 @@ if [ -e package.json ]; then
 
     npm ls --production --parseable 2>&1 >/dev/null | \
       sed -n -e 's/^npm ERR! peer dep missing: \(.*\),.*/\1/p' | \
-        xargs -I{} echo -n '"{}" ' | \
-          xargs -I{} npm install --no-package-lock --no-save "{}"
+          xargs -I{} echo -n '"{}" ' | \
+            xargs -S16383 npm install --no-package-lock --no-save
```

Comment:
comment: I don't think the -S flag is required now that the template flag (`-I`) has been removed

Reply:
yes it should be fine without it now

Reply:
Have removed it again 🙂